index.js

```diff
@@ -43,10 +43,13 @@ module.exports = {
 
     '@typescript-eslint/brace-style': ['error', '1tbs', { allowSingleLine: true }],
     '@typescript-eslint/comma-dangle': ['error', 'never'],
+    '@typescript-eslint/consistent-type-definitions': ['off'],
     '@typescript-eslint/consistent-type-imports': ['error', {
       prefer: 'no-type-imports'
     }],
     '@typescript-eslint/no-empty-interface': 'off',
+    'no-unused-vars': ['off'],
```

Comment:
```suggestion
    'no-unused-vars': 'off',
```

If it's just `off`, maybe it doesn't have to be an array like others.

Reply:
Thanks, and yea that makes sense. Though one could argue any number of the rules could change at the dependency level. I do my best to track changes at that end so hopefully we can pick up any inconsistencies before upgrading (hence why we use a fixed dependency version). 👍